qdev_wrappers/dataset/doNd.py

```diff
@@ -37,6 +38,8 @@ def do0d(*param_meas:  Union[_BaseParameter, Callable[[], None]],
         The run_id of the DataSet created
     """
     meas = Measurement()
+    if write_period:
```

Comment:
I guess this will automatically protect the user from setting the write period to `0.0`, but perhaps in a slightly to "clever" way. Perhaps `if write_period is None` and an explicit handling of 0 is better?